mkdocs/docs/quick-start-demo.md

```diff
@@ -0,0 +1,183 @@
+
+These are the steps we will cover in this article:
+
+1. [Set up a Polygon ID wallet](#set-up-a-polygon-id-wallet)
+2. [Issue a new credential to attest to the ID Holder's attendance to the event](#issue-a-new-credential-to-attest-to-the-id-holders-age)
+3. [Fetch the newly created credential](#fetch-the-newly-created-credential)
+4. [Verify the credential validity](#verify-the-credential-validity)
+5. [Query the ID holder for specific information of his credential](#query-the-id-holder-for-specific-information-of-his-credential)
+
+## Set up a Polygon ID wallet
+As an Identity Holder, the individual who wants to have a credential to prove his age will need an application that can hold their credentials. In our case, we will be using the Polygon ID Wallet.
+
+To get started with the Polygon ID Wallet, download the Polygon ID Wallet App and create an Identity:
+
+- For Android: <a href="https://play.google.com/store/apps/details?id=com.polygonid.wallet" target="_blank">Polygon ID on Google Play</a>
+- For iOS: <a href="https://apps.apple.com/us/app/polygon-id/id1629870183" target="_blank">Polygon ID on the App Store</a>
+
+!!!note
+    Polygon ID wallet is an implementation of the Wallet SDK, as a way of showcasing its possibilities. Head to [the Polygon ID SDK documentation](./wallet/wallet-sdk/polygonid-sdk/polygonid-sdk-overview.md) to know more about how it works. 
+
+The process from downloading to creating an identity on the Polygon ID Wallet is just as it is shown below. You need to download the app, create a wallet, set up a PIN number and the wallet is ready to be used. 
+
+![Quick start demo wallet](./imgs/quick-start-demo-wallet.png)
+
+!!!warning
+    This demo is using Polygon's Mumbai testnet. Make sure this network is selected on the app settings.
+    <div align="center">
+        <img width="300" src="../imgs/quick-start-demo/settings-mumbai.jpeg"></img>
+    </div>   
+
+## Issue a new credential to attest to the ID Holder's event attendance
+A trusted entity, for instance, a private institution will now play the role of an issuer. It will be possible for creating the credential and send it to the ID Holder.
+We are using a testing environment to manage credentials: [https://issuer-ui.polygonid.me](https://issuer-ui.polygonid.me). This is the place where the trusted entity can create credentials, schemas and generate connections. 
+However, before you actually issue a credential using the Issuer node, you need to create a schema for that credential, which basically is the set of JSON files that gather all the attributes of that specific credential. 
+
+!!!note
+    To learn how to set up your own issuer environment by deploying an issuer node, visit the [Issuer section in the documentation](./issuer/issuer-overview.md).
+
+### Create a schema
```

Comment:
To be honest I don't know if this part of creating the schema is complicating it too much for a quick start, but not sure what you agreed with @mmilenkovic and the team. So I will let them comment on that.

Reply:
hey Otto, unfortunately, there was no other around. We needed to use a custom credential, so we had to come up with a new schema. I had the same feeling beginning too.